Adding a paragraph for encryption recommendations

gkomatsu opened the pull request:
New paragraph to guide customers to use Azure Encryptions to protect their data on the VMs. Clarification on the method to use is not enabling BitLocker directly on the OS.

gkomatsu committed:
Adding a paragraph for encryption recommendations

New paragraph to guide customers to use Azure Encryptions to protect their data on the VMs. Clarification on the method to use is not enabling BitLocker directly on the OS.

gkomatsu committed:
fixed URL - removed locale

Removed en-us locale form URL link

gkomatsu reviewed (commented):
Remove local from link

gkomatsu committed:
URL Link fix

gkomatsu committed:
link fix 2

gkomatsu reviewed (commented):
Fixing link

Heidilohr commented on articles/virtual-desktop/security-guide.md:
```diff
@@ -88,6 +88,10 @@ When choosing a deployment model, you can either provide remote users access to
 
+### Encrypt your VM
+
+Encrypt your VM with [managed disk encryptio options](../virtual-machines/disk-encryption-overview.md) to protect stored data from unauthorized access. 
```
```suggestion
Encrypt your VM with [managed disk encryption options](../virtual-machines/disk-encryption-overview.md) to protect stored data from unauthorized access. 
```

gkomatsu replied:
Thank you for pointing out the typo.

gkomatsu committed:
Fixing typo Encryption

gkomatsu committed:
Additional edits for Azure Disk Encryption

Clarification on BitLocker enablement is recommended using Azure Disk Encryption in vTPM section.
Added Windows 11 Enterprise and Enterprise multi-session to the supported list on nested virtualization and Windows Defender Application Control.

gkomatsu reviewed (commented):
Additional edits to clarify BitLocker enablement using Azure Disk Encryption
